netpyne_ui/netpyne_geppetto.py

```diff
@@ -331,6 +331,16 @@ def getNetPyNELFPLocationsPlot(self):
             svgs.append(ui.getSVG(fig[0][0]))
         
         return svgs.__str__()
+
+    def getNetPyNERxDConcentration(self):
```

Comment:
@salvadord Can we name this method following the standard, i.e. 'getNetPyNERxDConcentrationPlot'?

Reply:
yes, can rename to that